spec/descriptions/tagInfrastructureAnalyze.md

```diff
@@ -4,6 +4,10 @@ It is also possible to [search and filter entities](#operation/getEntities) and
 
 ## Important notes
 
+### Prerequisites
+
+For self-hosted installations, Beeinstana is required for this endpoint group.
```

Comment:
- Should be BeeInstana instead of Beeinstana.
- It would be good to reference the self-hosted install section that tells how to enable BeeInstana: https://www.ibm.com/docs/en/instana-observability/current?topic=premises-installing-instana-backend-docker#beeinstana-metric-pipeline-beta

Reply:
Thanks I've made these changes